tests/jq.test

```diff
@@ -1788,6 +1788,19 @@ try -. catch .
 "very-long-string"
 "string (\"very-long-...) cannot be negated"
 
+try (.-.) catch .
+"very-long-string"
+"string (\"very-long-...) and string (\"very-long-...) cannot be subtracted"
+
+"x" * range(0; 12; 2) + "☆" * 5 | try -. catch .
```

Comment:
Would step size 1 test more `jvp_utf8_backtrack` cases?

Reply:
I don't think so. The current test case covers code enough IMO.